tests/cupy_tests/indexing_tests/test_insert.py

```diff
@@ -7,3 +7,31 @@
 class TestInsert(unittest.TestCase):
 
     _multiprocess_can_split_ = True
+
+    @testing.for_all_dtypes()
+    @testing.numpy_cupy_array_equal()
+    def test_fill_diagonal_shape1(self, xp, dtype):
+        a = testing.shaped_arange((3, 3), xp, dtype)
+        xp.fill_diag(a, val=0)
```

Comment:
`fill_diag` may be typo of `fill_diagonal`?

Please specify `accept_error=False` for `testing.numpy_cupy_array_equal` unless you actually intend to accept errors in test code. Otherwise, errors such as typo are just ignored and tests would still pass (see https://github.com/pfnet/chainer/issues/1419.)